### ISSUE TITLE
Accessibility Test Plan / Handling for new OS (Apple and Android)

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#9099]
**Description:** 
As Accessibility, I want to prepare for future testing of new operating system changes from Mobile providers for accessibility so that I am best prepared when it comes and also have a process to identify what should be tested and what I need to do when reviewing new OS changes that may be relevant . 

**Notes:** 

**AC**
